bliss/modules.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from yolov5.models.yolo import autopad
+
+
+class Conv3d(torch.nn.Module):
+    """Wrapper for torch.nn.conv3d layer.
+
+    Args:
+        c1: input channels
+        c2: output channels
+        k: kernel size
+        s: kernel stride length
+        p: input padding amount
+        g: controls connections between inputs and outputs
+        d: kernel dilation value
+        act: torch.nn activation function
+    """
+
+    default_act = torch.nn.SiLU()  # default activation
+
+    def __init__(self, c1, c2, k=1, s=1, p=None, g=1, d=1, act=True):
+        super().__init__()
+        self.conv = torch.nn.Conv3d(
+            c1, c2, k, s, autopad(k, p, d), groups=g, dilation=d, bias=False
+        )
+        self.bn = torch.nn.BatchNorm3d(c2)
+        self.act = (
+            self.default_act
+            if act is True
+            else act  # noqa: WPS509
+            if isinstance(act, torch.nn.Module)
+            else torch.nn.Identity()
```

Comment:
This if-else-if-else block is hard to reach, although it's nice that the variable your assigning to (`self.act`) only has to appear once. I wonder if there's a clearer way to write it.

Reply:
Can you replace `if act is True` with `if act`?

Reply:
This is actually also modified yolo code - I'm sure there's a cleaner way to write it without the coverage issue

Reply:
I see, it's fine as if it's just copied code -- but please put a comment next to it saying where the code came from